lib/libcam/tests/libcam_test.c

```diff
@@ -0,0 +1,170 @@
+#include <sys/cdefs.h>
```

Comment:
Missing licensing above. Suggested licensing: https://svnweb.freebsd.org/base/head/lib/libc/tests/sys/queue_test.c?revision=290909&view=markup#l1 (line 1-25 -- I'd just take it verbatim ;)..).